### PR TITLE
setup BenchmarkCI (changes on top of #148)

### PR DESCRIPTION
**Pull request by tkf**
changes on top of #148

### CODE REVIEW

**Comment by jw3126** [comment]
Shall we merge it?

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      JULIA_DEBUG: BenchmarkCI
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: 1.3
+      - name: Install dependencies
+        run: julia -e 'using Pkg; pkg"add PkgBenchmark https://github.com/tkf/BenchmarkCI.jl"'
+      - name: Run benchmarks
+        run: julia -e 'using BenchmarkCI; BenchmarkCI.judge(baseline = "HEAD")'
```
Comment (jw3126) [inline .github/workflows/benchmark.yml]:
```suggestion
        run: julia -e 'using BenchmarkCI; BenchmarkCI.judge()'
```

**Comment by johnnychen94** [comment]
I'll do that when https://github.com/JuliaRegistries/General/pull/8348 is merged

File: .github/workflows/benchmark.yml (hunk 1 above)
Comment (johnnychen94) [reply .github/workflows/benchmark.yml]: Do we need to commit this? @tkf

Comment (tkf) [reply .github/workflows/benchmark.yml]: Yes, it was for build passing even when origin/master (default `baseline`) is not ready for benchmark.

**Pull request merged**